lib/assets/chart_builder/main.css

```
@@ -6,61 +6,33 @@
   --gray-50: #f8fafc;
   --gray-100: #f0f5f9;
   --gray-200: #e1e8f0;
+  --gray-300: #cad5e0;
   --gray-400: #91a4b7;
   --gray-500: #61758a;
   --gray-600: #445668;
   --gray-800: #1c2a3a;
-}
-
-.row {
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  align-content: stretch;
-  flex-wrap: nowrap;
-}
 
-.row > *:not(:first-child) {
-  margin-left: 8px;
+  --primary-100: #ecf0ff;
```

Just so it matches Livebook :)

```suggestion
  --blue-100: #ecf0ff;
```